src/shared/loop-util.c

```diff
@@ -414,6 +418,11 @@ int loop_device_make(
                         return -EBUSY;
 
                 loopdev = mfree(loopdev);
+
+                /* Wait some random time, to make collision less likely. Let's pick a random time in the
+                 * range 0ms…250ms, linearly scaled by the number of failed attempts. */
+                (void) usleep(random_u64() % (UINT64_C(10) * USEC_PER_MSEC +
```

Comment:
USEC_PER_MSEC already has the necessary type, so the UINT64_C() cast is a noop. (Though it doesn't hurt.)